code/modules/clothing/under/miscellaneous.dm

```diff
@@ -691,12 +691,18 @@
 		if(extinguishes_left)
 			if(next_extinguish > world.time)
 				return
-			next_extinguish = world.time + extinguish_cooldown
-			extinguishes_left--
-			H.visible_message("<span class='warning'>[H]'s suit automatically extinguishes [H.p_them()]!</span>","<span class='warning'>Your suit automatically extinguishes you.</span>")
-			H.ExtinguishMob()
-			new /obj/effect/particle_effect/water(get_turf(H))
-	return 0
+				next_extinguish = world.time + extinguish_cooldown

```

Comment:
This looks like it's on the wrong indentation level

Reply:
I mean it works.

I'm not exactly an ace coder here I just do what works.

Reply:
It "works" but you removed the cooldown and limited uses

Reply:
No I didn't, actually. They're still there.

Reply:
I'll change it in a minute give me a second.

Reply:
I think @spacemaniac has a point, this code shouldn't get run because of the return on line 693, and the code you changed had this line under indentation level 3, not 4

Reply:
Mate I'm changing it, I just said.